File: src/Rent.java
import java.time.LocalDate;
public class Rent {
    private  Car car;
    private  Customer customer;
    private  LocalDate rentalDate;
    private  int rentalDuration;

    public Rent(Car car,Customer customer, LocalDate rentalDate, int rentalDuration){
        this.car=car;
        this.customer=customer;
        this.rentalDate=rentalDate;
        this.rentalDuration=rentalDuration;
    }

    public Car getCar() {
        return car;
    }

    public Customer getCustomer() {
        return customer;
    }

    public LocalDate getRentalDate() {
        return rentalDate;
    }

    public int getRentalDuration() {
        return rentalDuration;
    }
}

File: Rent.cs
using System;

public class Rent
{
    public Car Car { get; }
    public Customer Customer { get; }
    public DateTime RentalDate { get; }
    public int RentalDuration { get; }

    public Rent(Car car, Customer customer, DateTime rentalDate, int rentalDuration)
    {
        Car = car;
        Customer = customer;
        RentalDate = rentalDate;
        RentalDuration = rentalDuration;
    }
}